app/tests/unit/components/Layout/Footer.test.tsx

```diff
@@ -0,0 +1,14 @@
+import Footer from "components/Layout/Footer";
+import { render, screen } from "@testing-library/react";
+import footerLinks from "data/dashboardLinks/footerLinks";
+
+describe("The Footer Component", () => {
+  it("Renders footer links", () => {
+    render(<Footer />);
+    footerLinks.forEach(({ name, href }) => {
```

Comment:
We want the test to break if the content of the `footerLinks` changes, so I'd recommend hard-coding the expectations here